src/style.css

```
@@ -0,0 +1,110 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 14px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  padding: 16px 5px 0;
+  height: 134px;
+  width: 160px;
+}
+
+.card__descr {
```

```suggestion
.card__description {
```
don't  make shorten names